src/core/stompest/protocol/parser.py

```diff
@@ -64,6 +69,8 @@ def add(self, data):
         
         :param data: An iterable of characters. If any character evaluates to :obj:`False`, that stream will no longer be consumed.
         """
+        if version_info[0] == 3 and data.__class__.__name__ not in ['str', 'generator']:
```

Comment:
Don't refer to class names. You should use either duck-typing or `isinstance` or some other polymorphic mechanism.